app/components/report-wrong-data.js

```diff
@@ -54,6 +54,10 @@ export default class ReportWrongDataComponent extends Component {
       this.association ? this.association.name : ''
     } %0D%0A vCode: ${
       this.vCode
-    } %0D%0A Aan te passen op het ${url} (Verenigingsloket), pagina 'Mijn gegevens'.`;
+    } %0D%0A Aan te passen op het ${url} (Verenigingsloket), pagina 'Mijn gegevens'.
+      %0D%0A ----------------------------------------------------------------------
```

Comment:
We should probably reduce the amount of `-` characters? In one of the Jira issues I read that outlook has a limit, but I can't test it myself :thinking:

Reply:
I've reduced the amount and tested it in Outlook Web, and it seems to work there. I can't test the desktop app though.